packages/db-dashboard/src/components/metrics/Metrics.jsx

```diff
@@ -18,6 +18,7 @@ export default function Metrics () {
       }
     }
     getMetrics()
+    setInterval(getMetrics, 5000)
```

Comment:
I think this might plat havoc with the tests? Or maybe not? Don't we need to ever stop it?

Reply:
From the dashboard, if we want to stop it we should add some ui control. For instance to enable refresh, and with which interval. IMO we can add this also later (even if is not much work, but we might reason a little about the ui design). About the test, we don't have unit tests for this components :(.  So nothing should happen at all.